EDDI/EDDI.cs

```diff
@@ -1479,7 +1479,7 @@ private bool eventFSDEngaged(FSDEngagedEvent @event)
         private bool eventFSDTarget(FSDTargetEvent @event)
         {
             // Set and prepare data about the next star system
-            NextStarSystem = StarSystemSqLiteRepository.Instance.GetOrCreateStarSystem(@event.system);
+            NextStarSystem = StarSystemSqLiteRepository.Instance.GetOrFetchStarSystem(@event.system);
```

Comment:
I think this should remain `GetOrCreateStarSystem`. We're pre-loading data for use on the next jump, but that's not useful if we're not saving the data as we retrieve it. The star system name is coming directly from the journal so we can trust that it's a real star system.